packages/custom-webpack/README.md

```diff
@@ -154,7 +154,7 @@ Builder options:
 ```js
 "architect": {
   ...
-  "build": {
+  "serve": {
```

Comment:
Is this possible you confused `serve` with `server`? Because the right builder for the `serve` target is `custom-webpack:dev-server` (as the documentation states).  
`custom-webpack:server` is for SSR flow and it should be inside `build` architect target.

Reply:
Thanks